pkg/util/version/const.go

```diff
@@ -41,7 +41,7 @@ var DefaultInstallStream = Stream{
 // FluentbitImage contains the location of the Fluentbit container image
 func FluentbitImage(acrDomain string) string {
 	// https://github.com/microsoft/azurelinux/releases
-	return acrDomain + "/fluentbit:1.9.10-cm20240628@sha256:1d0b1789f4eb06f5c74fa5def306bd5095852882f63d91578ef589e8eab18765"
+	return acrDomain + "/fluentbit:1.9.10-cm20240628@sha256:2a8819d6a6500fbd5c2bb3bb2d0cbffbe842e3fdf827130c267be7cc5ee51ced"
```

Comment:
I am getting an error retrieving this digest:
```
$ podman pull --creds *** arosvc.azurecr.io/fluentbit:1.9.10-cm20240628@sha256:2a8819d6a6500fbd5c2bb3bb2d0cbffbe842e3fdf827130c267be7cc5ee51ced
Trying to pull arosvc.azurecr.io/fluentbit@sha256:2a8819d6a6500fbd5c2bb3bb2d0cbffbe842e3fdf827130c267be7cc5ee51ced...
Error: initializing source docker://arosvc.azurecr.io/fluentbit@sha256:2a8819d6a6500fbd5c2bb3bb2d0cbffbe842e3fdf827130c267be7cc5ee51ced: reading manifest sha256:2a8819d6a6500fbd5c2bb3bb2d0cbffbe842e3fdf827130c267be7cc5ee51ced in arosvc.azurecr.io/fluentbit: manifest unknown: manifest sha256:2a8819d6a6500fbd5c2bb3bb2d0cbffbe842e3fdf827130c267be7cc5ee51ced is not found
```

Reply:
Corrected it... this is the checksum I originally had, but I changed it to the value I saw in arointsvc. Updating arointsvc again.
```bash
teven@rh-thinkpad-p1:~/go/src/github.com/Azure/ARO-RP$ podman pull --authfile ~/arosvc.dockerconfig arosvc.azurecr.io/fluentbit:1.9.10-cm20240628@sha256:f2bfd9bef5eb1ab6ee580b585b3ef1280386c919b12901b7d7127de60c60e7e5
Trying to pull arosvc.azurecr.io/fluentbit@sha256:f2bfd9bef5eb1ab6ee580b585b3ef1280386c919b12901b7d7127de60c60e7e5...
Getting image source signatures
Copying blob ac74d26ce7d7 skipped: already exists  
Copying blob bbeff14dd44c skipped: already exists  
Copying blob 2181929f18e3 skipped: already exists  
Copying blob 13f9584a5f5b skipped: already exists  
Copying config f0b9fcf31a done   | 
Writing manifest to image destination
f0b9fcf31ac1d6031bdb5aba599752980ed04b6e883796e5a865c46a2f84d1c3
```

Reply:
I see what happened. I took the checksum from the arointsvc web portal, which is incorrect. It's not showing the correct digest.